lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -129,6 +129,17 @@ describe('CategoryRenderer', () => {
     assert.ok(warningEl.textContent.includes(auditResult.warnings[1]), '2nd warning provided');
   });
 
+  it('expands warning audit group', () => {
```

Comment:
nit: move this under the `clumping` `describe()` since it's really about the clump correctly working